Modules/errnomodule.c

```diff
@@ -46,24 +46,41 @@ static PyMethodDef errno_methods[] = {
 
 /* Helper function doing the dictionary inserting */
 
-static void
-_inscode(PyObject *d, PyObject *de, const char *name, int code)
+static int
+_add_errcode(PyObject *d, PyObject *de, const char *name, int code)
```

Comment:
Hum, what do you think of renaming variables?

```suggestion
_add_errcode(PyObject *module_dict, PyObject *error_dict, const char *name_str, int code_int)
```

Also rename `u` to `name` and `v` to `code`, to be able to write below:

```
PyDict_SetItem(module_dict, name, code)
```